docs/production/requirements.md

```diff
@@ -29,12 +29,13 @@ can't support you, but
 
 #### Operating System
 
-Ubuntu 18.04 Bionic, Ubuntu 16.04 Xenial, Debian Buster and Debian
-Stretch are supported for running Zulip in production.  64-bit is
-recommended.  We also recommend installing on the newest option you're
-comfortable with, to save your organization the work of upgrading
-(Ubuntu Trusty [reached end of life in April 2019][trusty-eol]; Zulip
-2.0 was the last major release to support it).
+Ubuntu 18.04 Bionic, Debian Busterm, Ubuntu 16.04 Xenial (deprecated),
```

Comment:
typo: "Busterm"

Also maybe "Debian 10 Buster"? Some public-facing Debian materials will say just the number and not the codename, as confusing as that is when so much else never mentions the number. (And I had to look up the number just now.)

Similarly "Debian 9 Stretch".